### ISSUE TITLE
Hand over gitbook-plugin-mermaid to GitbookIO

### ISSUE THREAD

**Issue by Soreine** [JozoVilcek/gitbook-plugin-mermaid#27]
Hi @JozoVilcek,

At @GitbookIO, we have noticed a fragmentation of the mermaid plugins for gitbook (with varying quality) https://plugins.gitbook.com/browse?q=mermaid

We would like to maintain an official mermaid plugin, and you own the name `gitbook-plugin-mermaid`. The activity on your plugin seems to slow down. We offer to maintain the plugin for you. In exchange we will keep a proper work attribution, and give appropriate credit to you, as the original author of the plugin.

Let me know if you're interested :)

**Comment by JozoVilcek** [comment]
Yes, that is correct. I do not have focus nor capacity keep this project properly synchronized with gitbook. If you are willing to take it under GitbookIO wings, yes, by all means, I will be more than happy to hand it over.

**Comment by llwt** [comment]
Any movement on this? I'd love to see a singe mermaid GB plugin.

**Comment by JozoVilcek** [comment]
I have added @Soreine as collaborator for npmjs project. I am happy to do anything necessary to hand it over ...
